pkg/exporter/azureblob_exporter.go

```diff
@@ -32,48 +33,8 @@ func NewAzureBlobExporter(creationTime, hostname string) *AzureBlobExporter {
 	}
 }
 
-func createContainerURL() (azblob.ContainerURL, error) {
```

Comment:
💡 How these changes relates with the Kind cluster support which this PR is intending? Is this just moved below or what exactly is changed.